src/main/java/org/broadinstitute/hellbender/engine/AssemblyRegion.java

```diff
@@ -166,6 +166,10 @@ public boolean isActive() {
         return isActive;
     }
 
+    void setIsActive(final boolean value) {
```

Comment:
Add javadoc for this method, and explain why it's package-private (eg., "Changing the isActive state after construction is a debug-level operation that only engine classes like AssemblyRegionWalker should be able to do")

Reply:
Done